testcases/open_posix_testsuite/scripts/locate-test

```diff
@@ -85,7 +85,7 @@ fi
 
 case "$mode" in
 buildonly)
-	find "$WHERE" -type f -name "*.c" | grep buildonly
+	find "$WHERE" -type f -name '*.c' | grep buildonly
```

Comment:
On a release build, this doesn't make any difference. During development though, the developer may have the wrong file in the wrong place and actually get a surprising result during the build.